test/unit/annotation_spec.js

```diff
@@ -1659,6 +1685,64 @@ describe("annotation", function () {
         done();
       }, done.fail);
     });
+
+    it("should get rendered annotation for printing", function (done) {
+      buttonWidgetDict.set("V", Name.get("1"));
```

Comment:
Since we're not testing the field value in this test, we might not even need to set it.